controllers/chat/sendSignedMessage.js

```diff
@@ -170,20 +170,18 @@ const sendSignedMesage = async (req, res) => {
       dataPayload.attachment = '';
     }
 
-    channelMember.forEach((member) => {
-      let payload = {
+    BetterSocialCore.fcmToken.sendChatNotificationByChannelMembers(
+      channelMember,
+      {
         notification: notificationPayload,
-        data: {
-          ...dataPayload,
-          is_annoymous: member.is_anonymous.toString()
-        }
-      };
-
-      BetterSocialCore.fcmToken.sendChatNotification(member.user_id, payload, {
+        data: dataPayload
+      },
+      {
         priority: 'high',
         content_available: true
-      });
-    });
+      }
+    );
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The new implementation of sending notifications to channel members seems more efficient as it sends notifications in a batch rather than individually. However, the `is_anonymous` field is missing in the new data payload. If this field is still required, you should add it back.
```diff
     BetterSocialCore.fcmToken.sendChatNotificationByChannelMembers(
       channelMember,
       {
          notification: notificationPayload,
+         data: {
+           ...dataPayload,
+           is_annoymous: member.is_anonymous.toString()
+         }
       },
       {
          priority: 'high',
          content_available: true
       }
     );
```
Please ensure that the `sendChatNotificationByChannelMembers` function handles the `is_anonymous` field appropriately.


<!-- This is an auto-generated comment by OSS CodeRabbit -->